index.html

```diff
@@ -0,0 +1,117 @@
+                    </p>    
+                </div>
+                <div class="date">
+                    <h3>2022.01.03(MON) ~ 8(SAT)</h3>
+                    <p>@ S.S.A.M.B empowerment of West and Central Africa</p>
+                </div>
+            </section>
+            <section class="section-2">
+                <p class="main-p">Main Program</p>
+                <hr>
+                <div class="cards">
+                    <div class="card1">
+                        <div class="index">
+                        <img class="fas" src="images/icons8-picture-64.png" alt="picture description">
+                        </div>
+                    <h5>About</h5>
+                    <P>SSAMB is made up of 5 talented and career driven youths whose aim is to empower young black girls into the tech industry</P>
+                    </div><br>
+                    <div class="card1">
+                        <img class="fas" src="images/icons8-picture-64.png" alt="image description">
+                        <h5>Eligible</h5>
+                        <p>All black girls aspiring to get into the field are highly encourage to take advantage of this opportunity.</p>
+                    </div><br>
+                    <div class="card1">
+                        <img class="fas" src="images/icons8-box-64.png" alt="images">
+                        <h5>Career</h5>
+                        <p>The community makes it flexible for you to pick out any tech field of your. we train you from zero to hero.</p>
+                    </div><br>
+                    <div class="card1">
+                        <img class="fas" src="images/icons8-uncheck-all-64.png" alt="imagess">
+                        <h5>Sponsors</h5>
+                        <p>The SSAMB community has partnership with other cohort of tech movement You don't have to border about payment.</p>
+                    </div><br>
+                    <div class="card1">
+                        <img class="fas" src="images/icons8-settings-64.png" alt="imagesss">
+                        <h5>FAQs</h5>
+                        <p>If you have gone throuh the programs requirement and have random questions, you can explore our FAQs for more clarity.</p>
+                    </div><br>
+                </div>
+                <button class="register-btn" type="button">Click here to register</button>
+                <a class="prog" href="#">SEE THE PROGRAM</a>
+            </section>
+            <section class="section3">
+                 <p>Guest Speakers and host</p>
+                 <hr class="newhr">
+            <div class="speaker-container">
+
+            </div>
+            </section>
+            
+  <section class="last--section">
+    <div class="footer">
+        <p>partner</p>
+        <hr>
+        <div class="grid-container">
+          <h3>yogaya Inc</h3>
+          <h2>DataGend</h2>
+          <h2>Mozilla</h2>
+          <h1>AirDrops</h1>
```

Comment:
[OPTIONAL]

- Nice use of the heading tags. 👍 However, kindly note that the heading(h1-h6) tags have a hierarchy that should be followed. `h1` tags should be used for very important text, such as the name of the app, and just one `h1` tag should be used on a page. I hope this makes sense 😀 

To learn more, please, go through the [W3schools tutorial](https://www.w3schools.com/html/html_headings.asp) on headings.